deploy/registry/local/ingress.yaml

```diff
@@ -11,11 +11,12 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: che-plugin-registry
+  namespace: operators
```

Comment:
Why do you force installing the plugin registry in the `operators` namespace ?

Afair this namespace doesn't exist on OpenShift.

Reply:
Let's discuss alternatives in https://github.com/che-incubator/che-workspace-crd-operator/pull/3#discussion_r363653290